src/rest/subhandlers/rest_basic_queries.erl

```diff
@@ -11,9 +11,8 @@
 %% API
 -export([
     init/2,
-    content_types_provided/2,
-    content_types_accepted/2,
     allowed_methods/2,
+    content_types_accepted/2,
```

Comment:
Konwencja nazewnictwa nadal jest nie spójna

Reply:
Hmm... zapamiętam sobie tę uwagę o nie zamykaniu konwersacji - tymczasem powtórzę to, co najwyraźniej cię nie wyświetliło w tym co zamknałem:

funkcja content_types_accepted to jest obowiązkowy callback cowboy-a, nie można go sobie zmienić, bo każdy handler obsługujący POST/PUT/PATCH musi to implementować

Reply:
A co do tego poniżej (rest_meta_info.erl) ok, niech będzie. Zdaję sobie sprawę że tak jak teraz jest nieeleganckie, ale po prostu mi się nie chciało poprawiać takiej pierdółki. No cóż, zrobi się